README.md

```diff
@@ -4,6 +4,8 @@ If you are unfamiliar with PerfView, there are [PerfView video tutorials](http:/
 As well as [Vance Morrison's blog](http://blogs.msdn.com/b/vancem/archive/tags/perfview) which also gives overview and getting 
 started information. 
 
+[![Build status](https://ci.appveyor.com/api/projects/status/fxtu3xa874whk2w0?svg=true)](https://ci.appveyor.com/project/sharwell/perfview)
```

Comment:
This is fine.    The one thing that would be good to change is to move the whole thing to right before the 'Building PerfView yourself'.    I realize that it makes finding the status a bit harder to find, but I also think I does make the most sense there (before this it was about using PerfView, after it is a building it yourself, this section is half way (getting daily builds).